Sources/TuistCache/Cache/URL+Cache.swift

```diff
@@ -3,27 +3,41 @@ import Foundation
 extension URL {
```

Comment:
I think I've mentioned it in the past, but I don't think the responsibility of the methods below should include throwing if the URL is valid. That's something I think should be validated earlier in the process, and at this point, you should assume that the URL is right and therefore remove all the throws.

Reply:
Ok, I removed that assertion and the associated logic